### ISSUE TITLE
Digest size

### ISSUE THREAD

**Issue by philiprbrenan** [BLAKE3-team/BLAKE3#107]
Please state the digest size some where near the front of your description?  Thanks!

**Comment by oconnor663** [comment]
> The default output size is 256 bits.

That sentence is near the top of the readme. Do you think it needs to be more prominent?

**Comment by philiprbrenan** [comment]
Yes please. I see the width now that I know the number to look for, but prior to this, I had skimmed the README several times and not been able to locate the width - which was the most important piece of information to me at that point in time.  A feature of Blake* that I find very useful is that you can create a digest of any width that is a multiple of 8 bits - this is very helpful to new users, like myself, who, might find dealing with 256 bit hashes somewhat intimidating and would prefer something smaller to start with.

**Comment by oconnor663** [comment]
> prefer something smaller to start with

Could you clarify that part for me? Keeping the default 256-bit output length is necessary for [collision resistance](https://en.wikipedia.org/wiki/Collision_resistance). Some applications don't require collision resistance, but distinguishing between the ones that do and the ones that don't is tricky. In terms of what we should encourage in the readme, I'd much rather encourage everyone to just use the default length.

**Comment by philiprbrenan** [comment]
Perhaps something like:

Width: 256 bits for cryptographic applications else in multiples of 8 bits.

**Comment by cryptoquick** [comment]
I think this could be expressed in the bullet list up top:
- **Configurable digest size**, with a default of 256 bits for cryptographic applications. Configured in multiples of 8 bits.
